### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by zizdepski**

### CODE REVIEW

File: client/src/components/pages/Calendar/Calendar.jsx [hunk 1]
```diff
@@ -1,9 +1,58 @@
...
+/* Each cell in calendar table will have a dropdown menu that
+displays workout plans that can be added to that day. */
 function Calendar() {
   return (
     <div>
       <h2>Calendar</h2>
+      <Table striped bordered hover>
+      <thead>
+    <tr>
+      <th>Monday</th>
```
Comment (copelka) [inline client/src/components/pages/Calendar/Calendar.jsx:13]: keep code DRY

File: client/src/components/pages/Calendar/Calendar.jsx [hunk 2]
```diff
@@ -1,9 +1,58 @@
...
+      <th>Wednesday</th>
+      <th>Thursday</th>
+      <th>Friday</th>
+      <th>Saturday</th>
+      <th>Sunday</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><DropdownButton id="dropdown-basic-button" title="Dropdown button" size="sm" variant="info"
```
Comment (copelka) [inline client/src/components/pages/Calendar/Calendar.jsx:24]: keep code DRY - how can you rewrite this to be more dynamic?

File: server/index.js [hunk 3]
```diff
@@ -3,8 +3,8 @@ const http = require('http');
 const express = require('express');
 const app = require('./app');
 const db = require('./db/index');
-const PORT = 3000;
 
+const PORT = 3000;
```
Comment (copelka) [inline server/index.js:7]: `const PORT = process.env.PORT || 3000` will give you the option to change the port in the env (easy for deployment)

Comment (zizdepski) [reply server/index.js:7]: Thanks, I'll get that tossed in.

**Pull request merged**